plenoptic/tools/signal.py

```diff
@@ -328,8 +251,8 @@ def rectangular_to_polar(real, imaginary):
     TODO: torch.abs(cmplx) torch.angle(cmplx)
```

Comment:
this docstring needs to be updated, since now it does operate on a complex tensor

Reply:
updated